tests/test_quickbooks_discovery.py

```diff
@@ -40,7 +39,7 @@ def test_run(self):
         self.assertTrue(all([re.fullmatch(r"[a-z_]+", name) for name in found_catalog_names]),
                         msg="One or more streams don't follow standard naming")
 
-        diff = self.expected_check_streams().symmetric_difference(found_catalog_names)
+        diff = expected_streams.symmetric_difference(found_catalog_names)
         self.assertEqual(len(diff), 0, msg="discovered schemas do not match: {}".format(diff))
         print("discovered schemas are OK")
```

Comment:
Please remove this print() statement and replace it with tap-tester logger.

Reply:
done